pkg/kv/dynamodb/store.go

```diff
@@ -219,7 +219,7 @@ func (s *Store) Get(ctx context.Context, partitionKey, key []byte) (*kv.ValueWit
 	var item DynKVItem
```

Comment:
I think you forgot one on line 216
(Can't add a comment on the exact line, as it is uncahnged)